sanic/request.py

```diff
@@ -212,9 +215,24 @@ def args(self):
         return self.parsed_args
 
     @property
-    def raw_args(self):
+    def raw_args(self) -> dict:
+        if self.app.debug:  # pragma: no cover
+            warnings.simplefilter("default")
+        warnings.warn(
+            "Use of raw_args will be deprecated in "
+            "the future versions. Please use args or not_grouped_args "
+            "properties instead",
+            DeprecationWarning,
+        )
         return {k: v[0] for k, v in self.args.items()}
 
+    @property
+    def not_grouped_args(self) -> list:
```

Comment:
I might be nitpicking but, I think it might be a good idea to rename this property to something more simpler like `query`/`args_as_tuple`/`arg_tuple` or something similar,

Reply:
The current properties are: 

- `args`
- `parsed_args`
- `raw_args`

My suggestion would be to follow this pattern and use either: `query_args` or `tuple_args`.

Reply:
+1 for `query_args`